### PR TITLE
Allow adding the body of the PR to the release

### PR DESCRIPTION
**Pull request by gavinlove**
We make extensive use of the pull request body and automatically send out the release body to a group of people when the release is deployed.

As such I would like to stop having to copy the PR body manually into the release notes. 

I have added a test however it is not very good because I think I need to be a PR in this [repo]( https://github.com/TimonVS/release-drafter-test-repo) with a body to do a proper test. If there is a way to get that or another way to add a test body please do let me know.

fixes #517

### CODE REVIEW

**Comment by jetersen** [comment]
Can you please revert all formatting changes?

**Comment by gavinlove** [comment]
Sure, although it was the pre-commit hook that made the changes. I will disable revert.

**Comment by jetersen** [comment]
I would prefer if those style changes are done in a separate pr

**Comment by gavinlove** [comment]
Makes sense!

File: lib/commits.js [hunk 1]
```diff
@@ -33,6 +33,7 @@ module.exports.findCommitsWithAssociatedPullRequestsQuery = /* GraphQL */ `
                   nodes {
                     title
                     number
+                    body
```
Comment (jetersen) [inline lib/commits.js:36]: let me regenerate after adding some body to those PRs :)

**Comment by jetersen** [comment]
of course the style PR introduced breaking changes.

**Comment by gavinlove** [comment]
Always the way :(

File: lib/commits.js (hunk 1 above)
Comment (jetersen) [reply lib/commits.js:36]: Perhaps the GraphQL can optionally only query for this if `change-template` includes `$BODY` thus reducing the amount of data required and reducing the time to response?

File: index.js [hunk 2]
```diff
@@ -40,6 +40,7 @@ module.exports = (app) => {
       context,
       branch,
       lastRelease,
+      config
```
Comment (jetersen) [inline index.js:43]: According to style it should be 😅 
```suggestion
      config,
```

**Comment by jetersen** [comment]
Hmm I want to add lint CI to save time with styling 😅

File: index.js (hunk 2 above)
Comment (gavinlove) [reply index.js]: Doh! yeah, I disabled the hook!

File: lib/commits.js (hunk 1 above)
Comment (gavinlove) [reply lib/commits.js]: Good call on this when I tested it I noticed the difference in the time to respond.

**Comment by jetersen** [comment]
@gavinlove I have added some bodies to the fixtures, can you try and regenerate?

**Comment by gavinlove** [comment]
Will do

**Pull request merged**